homework-10/script.js

```diff
@@ -0,0 +1,114 @@
+//Bai 1 : In bang nhan
```

Comment:
```suggestion
// Bai 1 : In bang nhan
```
Và không được dùng tiếng Việt
Có thể `quotes` lại đề bài

Reply:
Cái dấu cách này là từ bài 10 anh chưa chữa cho em nên em vẫn mắc nên ko tính là lỗi được đâu anh!